File: Java/src/Device.java
public class Device {
    private String deviceType;
    private int batteryLevel;


    public Device(String deviceType, int batteryLevel) {
        this.deviceType = deviceType;
        this.batteryLevel = batteryLevel;
    }

    public String getDeviceType() {
        return this.deviceType;
    }

    public void setDeviceType(String deviceType) {
        this.deviceType = deviceType;
    }

    public int getBatteryLevel() {
        return this.batteryLevel;
    }

    public void setBatteryLevel(int batteryLevel) {
        this.batteryLevel = batteryLevel;
    }





}

File: Python/Device.py
class Device:
    def __init__(self, device_type, battery_level):
        self.device_type = device_type
        self.battery_level = battery_level

    def get_device_type(self):
        return self.device_type

    def set_device_type(self, device_type):
        self.device_type = device_type

    def get_battery_level(self):
        return self.battery_level

    def set_battery_level(self, battery_level):
        self.battery_level = battery_level
